component/controller/SerialPortBridgeGeneric.cpp

```diff
@@ -85,6 +86,10 @@ void SerialPortBridgeGeneric::init()
 {
     checkConnection();
 
+    // To remove before v19.0 of the plugin
```

Comment:
:)

Reply:
Next time I will add the date of "creation", I think it's better.